zwave_js_server/model/notification.py

```diff
@@ -184,3 +184,47 @@ def status(self) -> PowerLevelTestStatus:
     def acknowledged_frames(self) -> int:
         """Return acknowledged frames property."""
         return self.data["args"]["acknowledgedFrames"]
+
+class MultilevelSwitchNotificationArgsDataType(TypedDict, total=False):
+    """Represent args for a Multi Level Switch CC notification event data dict type."""
+
+    event_type: int  # required
+    direction: str
+        
+        
+class MultilevelSwitchNotificationDataType(BaseNotificationDataType):
+    """Represent a Multi Level Switch CC notification event data dict type."""
+        
+    args: MultilevelSwitchNotificationArgsDataType  # required
+        
+        
+class MultilevelSwitchNotification:
+    """Model for a Zwave Node's Multi Level CC notification event."""
+        
+    def __init__(self, node: "Node", data: MultilevelSwitchNotificationDataType) -> None:
+        """Initialize."""
+        self.node = node
+        self.data = data
+        
+    @property
+    def node_id(self) -> int:
+        """Return node ID property."""
+        return self.data["nodeId"]
+        
+    @property
+    def command_class(self) -> int:
+        """Return command class."""
+        return self.data["ccId"]
+
+    @property
+    def event_type(self) -> int:
```

Comment:
We should create an `IntEnum` for `MultilevelSwitchCommand` in `zwave_js_server.const.command_class.multilevel_switch` and then return the enum value here instead of the raw int

Reply:
Do you mean for the event type?
ie., new enum in         return CommandClass.SWITCH_MULTILEVEL with something like this:

class MultilevelSwitchCommand(IntEnum):
	START_LEVEL = 4
	STOP = 5

Or for the Command classId?
return CommandClass.SWITCH_MULTILEVEL

Reply:
@raman325  see commit, I think this is what you meant.